packages/rollup/test/integration/far/a/b/c/BUILD.bazel

```diff
@@ -4,7 +4,7 @@ load("@npm_bazel_typescript//:index.bzl", "ts_project")
 package(default_visibility = ["//packages/rollup:__subpackages__"])
 
 ts_project(
-    srcs = ["index.ts"],
+    srcs = glob(["*.ts"]),
```

Comment:
Was this necessary for the fix? Or just as a better example for common use cases?

Reply:
Just a better explanation so that the ts_project srcs lines up with the tsconfig include